ada_2_fhir-r4/fhir/2_fhir_fhir_include.xsl

```diff
@@ -201,39 +202,67 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
         </xsl:choose>
     </xsl:function>
     
-    <!-- Outputs reference if input is ADA or fhirMetadata element -->
+    <!-- Outputs reference if input is ADA, fhirMetadata or ADA reference element -->
```

Comment:
Uitgebreid met de mogelijkheid om referenties in ADA te gebruiken (via het ADA-reference-mechanisme). Het aardige is dat dit buiten de driver om gaat; de ADA-instance definieert zelf de referentie.